extensions/amp-story/0.1/amp-story-base-layer.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Base layer from which other layers in a story page extend from.
+ */
+
+import {Layout} from '../../../src/layout';
+
+export class AmpBaseLayer extends AMP.BaseElement {
```

Comment:
In `buildCallback()` we can automatically add the CSS class name you created to the layer, with:

```javascript
this.element.classList.add('i-amphtml-story-layer');
```

And you'll need to make sure that `AmpStoryGridLayer` calls `super.buildCallback()` in its `buildCallback()` function.